pkg/storage/test/tuples.go

```diff
@@ -207,13 +207,44 @@ func TupleWritingAndReadingTest(t *testing.T, datastore storage.OpenFGADatastore
 		require.EqualError(t, err, expectedError.Error())
```

Comment:
Could you also add some tests to account for the comment above?

https://github.com/openfga/openfga/pull/999/files#r1366181079

Reply:
[a8cf564](https://github.com/openfga/openfga/pull/999/commits/a8cf564222df38400e3f6b4bcb6602bf6dd02079)